go/chat/convsource.go

```diff
@@ -932,6 +936,10 @@ func (s *HybridConversationSource) Clear(ctx context.Context, convID chat1.Conve
 	return s.storage.ClearAll(ctx, convID, uid)
 }
 
+func (s *HybridConversationSource) ClearLocalCache(ctx context.Context, convID chat1.ConversationID, uid gregor1.UID) error {
```

Comment:
What is the difference between this and `ClearAll`?

Reply:
seemed more efficient to call nuke rather than going through all of the messages of a conversation, maybe `ClearAll` should just call `MaybeNuke` instead of `clearUpthrough`

Reply:
I think that would be better, I'd rather not have two functions with effectively the same name in here.

Reply:
Although `ClearAll` has the advantage it actually removes the data right there. Maybe just call that, I don't think there is a real efficiency problem with calling it.

Reply:
oops just saw this, what do you mean by `removes the data right there`?

Reply:
All nuke does is take out the "block index", whereas `ClearAll` kills the messages too.

Reply:
ah gotcha